tests/time_tests/test_runner/test_timetest.py

```diff
@@ -17,29 +17,38 @@
 from pathlib import Path
 import logging
 import os
+import shutil
 
 from scripts.run_timetest import run_timetest
 from test_runner.utils import expand_env_vars
 
 REFS_FACTOR = 1.2      # 120%
 
 
-def test_timetest(instance, executable, niter, cl_cache_dir, test_info):
+def test_timetest(instance, executable, niter, cl_cache_dir, test_info, tmpdir):
```

Comment:
don't use `tmpdir` fixture because it doesn't clean up folder after test execution